### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NuriaComa**

### CODE REVIEW

File: staff/NuriaComa/js/challenges/scripts/CamelCase.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+console.log(">>JUNTAR PALABRAS")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/CamelCase.test.js]: use strict...

File: staff/NuriaComa/js/challenges/scripts/countWords.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+console.log(">>CONTAR PALABRAS")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/countWords.test.js]: use strict...

File: staff/NuriaComa/js/challenges/scripts/cube.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+"use strict";
+"use strict"
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/cube.js]: WTF DUDE.

File: staff/NuriaComa/js/challenges/scripts/cube.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+console.log(">>CUBE")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/cube.test.js:2]: use strict...

File: staff/NuriaComa/js/challenges/scripts/forEach.test.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+console.log(">>FOREACH")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/forEach.test.js:2]: use strict...

File: staff/NuriaComa/js/challenges/scripts/reduce.test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+console.log(">>REDUCE")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/reduce.test.js:2]: use strict...

File: staff/NuriaComa/js/challenges/scripts/roman.test.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/roman.test.js:2]: use strict...

File: staff/NuriaComa/js/challenges/scripts/toRandomCase.test.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
+console.log(">>MAYÚSCULAS ALEATORIAS")
```
Comment (agandia9) [inline staff/NuriaComa/js/challenges/scripts/toRandomCase.test.js]: use strict...

**Comment by manuelbarzi** [comment]
merging, but please, apply the pending changes anyway on next pr.

**Pull request merged**